### PR TITLE
Clean commands and scripts

### PR DESCRIPTION
**Pull request by InigoSJ**
Remove internal projects / names from commands and scripts. Modify READMEs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,34 @@
 
 This repository contains GCP connectors for Apache Flink.
```
Comment (clmccart) [inline README.md:3]: this isn't really true. this repo doesnt actually house any connectors (right now). it just houses examples for how to integrate GMF and OSS flink with GCP

File: README.md [hunk 2]
```diff
@@ -2,6 +2,34 @@
...
 
+## Examples
+
+You can find multiple examples that use GCP connectors on `flink-examples-gcp`.
```
Comment (clmccart) [inline README.md:7]: nit: in

File: README.md [hunk 3]
```diff
@@ -2,6 +2,34 @@
...
+
+These are:
+- Google Cloud Storage Load Generator
```
Comment (clmccart) [inline README.md:10]: let's label the pipelines as (Datastream API) or (Table API)

File: README.md [hunk 4]
```diff
@@ -2,6 +2,34 @@
...
+- Google Cloud Storage to Google Cloud Storage with Table API
+- Google Cloud Storage to Google Cloud Storage Unbounded WordCount
+- Google Cloud Storage to Google Cloud Storage Bounded WordCount
+- Google Managed Kafka Load Generator
+- Google Managed Kafka to BigQuery
+- Google Managed Kafka to Google Managed Kafka WordCount
+- Google Managed Kafka to Google Managed Kafka with Table API
+- PubSub Load generator
```
Comment (clmccart) [inline README.md:18]: nit: capitalize

File: README.md [hunk 5]
```diff
@@ -2,6 +2,34 @@
...
 This repository contains GCP connectors for Apache Flink.
 
+## Examples
+
+You can find multiple examples that use GCP connectors on `flink-examples-gcp`.
+
+These are:
+- Google Cloud Storage Load Generator
+- Google Cloud Storage to Google Cloud Storage with Table API
+- Google Cloud Storage to Google Cloud Storage Unbounded WordCount
```
Comment (clmccart) [inline README.md:12]: let's be remove the unbounded here and let it imply that streaming is the default and then for the bounded one let's remove the word "Bounded" and just label it as "(Batch)" instead

File: README.md [hunk 6]
```diff
@@ -2,6 +2,34 @@
...
+- Google Managed Kafka Load Generator
+- Google Managed Kafka to BigQuery
+- Google Managed Kafka to Google Managed Kafka WordCount
+- Google Managed Kafka to Google Managed Kafka with Table API
+- PubSub Load generator
+- PubSub to BigQuery WordCount
+
+## Scripts
+
+Contains scripts to create a Google Manage Flink deployment and start a Flink job.
```
Comment (clmccart) [inline README.md:23]: the official name if "BigQuery Engine for Apache Flink" instead of Google Managed Flink... i know...

File: README.md (hunk 1 above)
Comment (InigoSJ) [reply README.md:3]: Removed this

**Review by clmccart — APPROVED** [review]

**Pull request merged**